src/model/CollieConfig.ts

```diff
@@ -0,0 +1,49 @@
+import { Logger } from "../cli/Logger.ts";
+
+export class CollieConfig {
+  static CONFIG_FILE_PATH = ".collie/config.json";
+
+  static getFoundation(logger: Logger) {
+    try {
+      const config = JSON.parse(Deno.readTextFileSync(this.CONFIG_FILE_PATH));
+      const foundation = config["foundation"];
+      logger.tip(`Foundation ${foundation} is set in ${this.CONFIG_FILE_PATH}`);
+      return foundation;
+    } catch (e) {
+      if (e.name != "NotFound") {
```

Comment:
f: collie-cli always writes on stderr, stdout is reserved for data output

There are helpers in Logger class to print errors.
Also: wrap the error with a human readable message (failed to parse config file...), presenting users with technical errors and no info how to resolve is not good.